src/Command/AbstractCommand.php

```diff
@@ -106,4 +106,19 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
 
         return str_replace('.git', '', end($repositoryArray));
     }
+
+    /**
+     * Return the last packagist version.
+     */
+    final protected function getLastPackageVersion(Package $package): Package\Version
```

Comment:
Maybe "latest" is better than "last"?

Reply:
Can you rename the method accordingly and remove the comment? @VincentLanglet

Reply:
Sure, i will try to find time for this !